### PR TITLE
gh-570  Disabled signing

### PR DESCRIPTION
**Pull request by DmitryBespalov**
Disabled both confirm button and settings cell

### CODE REVIEW

File: Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift [hunk 1]
```diff
@@ -82,6 +62,33 @@ struct BasicAppSettingsView: View {
             self.trackEvent(.settingsApp)
         }
     }
+
+    @ViewBuilder
+    var signingWalletView: some View {
+        if signingKeyAddress != nil {
+            ForEach(0..<1) { _ in
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift:69]: I'm not going to refactor  it now, but was curious why there's ForEach to produce just 1 element?

File: Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift [hunk 2]
```diff
@@ -82,6 +62,33 @@ struct BasicAppSettingsView: View {
...
+                VStack(alignment: .leading, spacing: 4) {
+                    Text("Signing key").bold()
+                    AddressView(self.signingKeyAddress!)
+                }
+                .padding()
+            }
+            .onDelete { _ in
+                do {
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift:77]: This should be in the view model / model object of some sort

File: Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift (hunk 1 above)
Comment (sche) [reply Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift:69]: to enable swipe to delete

Comment (DmitryBespalov) [reply Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift:69]: wow!

**Review by sche — APPROVED** [review]

**Pull request merged**